README.md

```diff
@@ -65,13 +63,31 @@ The workflow takes the following inputs:
 1. `email_with_score` (optional & case-sensitive): Choose whether or not the e-mail sent out to participants will include the evaluation score or not. Can either be "yes" or "no". Defaults to "yes".
 1. `cpus` (optional): Number of CPUs to dedicate to the `RUN_DOCKER` process i.e. the challenge executions. Defaults to `4`
 1. `memory` (optional): Amount of memory to dedicate to the `RUN_DOCKER` process i.e. the challenge executions. Defaults to `16.GB`
+1. `scoring_container` (optional): The name of the container that the scoring script will be executed in
```

Comment:
Nit: consistency throughout this doc.
Some lists have a period at the end. Some do not.